From 3df49358c29266fce90ccbb54f67f9870da9b79e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Martin=20Storsj=C3=B6?= <martin@martin.st>
Date: Wed, 12 Apr 2023 11:04:22 +0000
Subject: [PATCH 16/23] [compiler-rt] [test] [profile] Add a marking that a
 test requires an MSVC target

This test uses lots of lld-link specific linker options that don't
work as such in mingw command lines.
---
 compiler-rt/test/profile/Windows/coverage-weak-lld.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/compiler-rt/test/profile/Windows/coverage-weak-lld.cpp b/compiler-rt/test/profile/Windows/coverage-weak-lld.cpp
index 1dc74cb22fa3..338be8e979c5 100644
--- a/compiler-rt/test/profile/Windows/coverage-weak-lld.cpp
+++ b/compiler-rt/test/profile/Windows/coverage-weak-lld.cpp
@@ -1,5 +1,6 @@
 // REQUIRES: lld-available
 
+// REQUIRES: target={{.*windows-msvc.*}}
 // RUN: %clang_profgen -fcoverage-mapping -c %s -o %t0.o
 // RUN: %clang_profgen -fcoverage-mapping -c %s -DOBJ_1 -o %t1.o
 // RUN: %clang_profgen -fcoverage-mapping -c %s -DOBJ_2 -o %t2.o
-- 
2.34.1

